flashlight/app/asr/tools/VoiceActivityDetection-CTC.cpp

```diff
@@ -63,7 +63,7 @@ const std::string kPerplexityPctSpeechExt = ".sts";
 
 using namespace fl::pkg::speech;
 using namespace fl::lib;
-using namespace fl::ext;
+using namespace fl;
```

Comment:
Not crazy about this, make the things in the top level namespace explicit if you feel like it